client/src/featureform/register.py

```diff
@@ -1342,6 +1365,15 @@ def register_training_set(self,
             owner = owner.name()
         if owner == "":
             owner = self.must_get_default_owner()
+        if resources and (features or label):
+            raise ValueError("Either a resource or features/label must be entered")
+        if isinstance(features,tuple):
+            raise ValueError("Features must be entered as a list")
+        if resources != "":
+            features = resources.features()
+            label = resources.label()
```

Comment:
Should check if label() has a label and if the label parameter has a label. Throw an error if both or neither have a label

Reply:
This should still check if they manually entered a label. 
Should be 
if resources.label() and label: error
if not resources.label() and not label: error